### PR TITLE
Highlight unread messages #1311

### LINKED ISSUE #1311: Highlight unread messages
Looking at the inbox at https://www.trustroots.org/messages, all messages look the same, no matter if they are read or unread.

Unread messages should be highlighted (e.g. bold & black instead of grey).

### PR DESCRIPTION
**Pull request by aldenfbrown**
## Proposed Changes

*  Added conditional that applies styling via a preexisting className

#### Testing Instructions

* Go to the inbox verify the unread messages appear bolded

Fixes #1311

### CODE REVIEW

**Comment by rumwerfer** [comment]
Works as expected.

File: modules/messages/client/components/InboxThread.js [hunk 1]
```diff
@@ -10,8 +10,14 @@ export default function InboxThread({ user, thread }) {
   const { t } = useTranslation('messages');
   const otherUser = findOtherUser(user, thread);
   const haveReplied = thread.userFrom._id === user._id;
+  const { read } = thread;
+
   return (
-    <li className="list-group-item threadlist-thread">
+    <li
+      className={`list-group-item threadlist-thread ${
+        !read ? 'threadlist-thread-unread' : ''
```
Comment (simison) [inline modules/messages/client/components/InboxThread.js:18]: We can use `classnames` package here to simplify; we use it elsewhere in codebase too.

**Comment by simison** [comment]
Can you have a look at the failure in CI?

File: modules/messages/client/components/InboxThread.js [hunk 2]
```diff
@@ -41,6 +47,7 @@ export default function InboxThread({ user, thread }) {
 InboxThread.propTypes = {
   user: userType.isRequired,
   thread: PropTypes.shape({
+    read: PropTypes.boolean.isRequired,
```
Comment (simison) [inline modules/messages/client/components/InboxThread.js:50]: I think it's `bool`, not `boolean`

Comment (aldenfbrown) [reply modules/messages/client/components/InboxThread.js]: Ahh that's right silly me I have been in TypeScript land too long. Hang tight.

Comment (aldenfbrown) [reply modules/messages/client/components/InboxThread.js]: Ok now we are good. Thanks!

**Pull request merged**